generators/client/templates/angular/src/test/javascript/spec/helpers/_routes.ts

```diff
@@ -0,0 +1,15 @@
+import { ActivatedRoute, Params } from '@angular/router';
+import { Observable } from 'rxjs';
+
+export class MockActivatedRoute extends ActivatedRoute {
```

Comment:
I already added a `MockActivatedRoute` in helpers/_activated-route.service.ts` could you remove that and use this in the entity specs as well. Alos rename this file to `route-helpers.ts`